test/test_interface_c.jl

```diff
@@ -0,0 +1,80 @@
+module TestInterfaceC
+
+using Test
+using Libdl
+using LibTrixi
+
+@testset verbose=true showtiming=true "C Interface" begin
+
+    # get path of shared object, relative to LibTrixi.jl
+    libtrixi_so_path = joinpath(dirname(pathof(LibTrixi)), "../../../../lib/")
+    push!(Libdl.DL_LOAD_PATH, libtrixi_so_path)
+
+    # try to load the shared object
+    libtrixi_handle = dlopen("libtrixi")
+
+
+    # TODO: get version and compare to something (?)
+    @testset verbose=true showtiming=true "Version info" begin
+        @test (@ccall "libtrixi".trixi_version_major()::Int64) == 0
+        @test (@ccall "libtrixi".trixi_version_minor()::Int64) == 1
+        @test (@ccall "libtrixi".trixi_version_patch()::Int64) == 0
+    end
+
+
+    # initialize julia from C
+    @ccall "libtrixi".trixi_initialize(dirname(Base.active_project())::Cstring, C_NULL::Ptr{Cvoid})::Cvoid
+
+    # TODO: test the simulation state handling
+    #       therefore a libelexir is loaded
+    libelixir_path = joinpath(dirname(pathof(LibTrixi)), "../examples/libelixir_tree1d_dgsem_advection_basic.jl")
```

Comment:
This means the test is based on just one example. (which is bad ?)

Alternatively one could construct a simstate mockup in a libelixir file. This could actually generate random pseudo simulation data (within reasonable bounds).
On the other hand, this approach would not test the real Trixi, OrdinaryDiffEq, ... workflow.